rust/datafusion/src/datasource/csv.rs

```diff
@@ -104,4 +105,8 @@ impl TableProvider for CsvFile {
             batch_size,
         )?))
     }
+
+    fn statistics(&self) -> Option<Statistics> {
+        unimplemented!()
```

Comment:
I think we should return `None` here to indicate that statistics are not available (for all the current data sources).

Reply:
OK ， Thanks andy, I'll fix it

Reply:
I thought that you would add statistics to the structs:

```rust
/// Table-based representation of a `ParquetFile`.
pub struct ParquetTable {
    path: String,
    schema: SchemaRef,
    statistics: Option<Statistics>,
}
```

and the `statistics` method would just return the `Option<Statistics>`:

```rust
    fn statistics(&self) -> Option<Statistics> {
        self.statistics.clone()
    }
```

and `statistics` would initially be instantiated as `None` until we implement it for each source:

```rust
        Ok(Self {
            path: path.to_string(),
            schema,
            statistics: None,
        })
```

I am new to rust but this is what you were thinking @andygrove ?

Reply:
Ok, I fix it.